exercise1/exercise8/Main.cpp

```diff
@@ -0,0 +1,27 @@
+/*Exercise 8 - Temperature
+Write a program that will ask for a temperature in Fahrenheit and convert it to Celsius, then ask for a temperature in Celsius and convert it to Fahrenheit
+Hint: 
+C = (F - 32.0) * 5.0 / 9.0
+F = C * 9.0 / 5.0 + 32.0
+*/
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	float f,f2, c,c2;
+	cout << "insert a temperature in Fahrenheit ";
+	cin >> f;
+	c2 = (f - 32.0) * 5.0 / 9.0;
```

Comment:
Floats should use a .f at the end of their value. So 32.0 is a double, but 32.0f is a float.